iceberg/iceberg-rest-server/src/test/java/org/apache/gravitino/iceberg/integration/test/IcebergRESTJdbcCatalogIT.java

```diff
@@ -68,6 +68,8 @@ public Map<String, String> getCatalogConfig() {
     configMap.put(
         IcebergConfig.ICEBERG_CONFIG_PREFIX + IcebergConfig.JDBC_INIT_TABLES.getKey(), "true");
 
+    configMap.put(IcebergConfig.ICEBERG_CONFIG_PREFIX + "jdbc.schema-version", "V1");
```

Comment:
should we enable this configuration by default, does it have some bad effect?

Reply:
This configuration will alter the metadata schema of JdbcCatalog. Perhaps it's preferable to allow users to set this explicitly? They should be aware of the exact changes made.

We could introduce a `jdbc.schema-version` property in IcebergConfig and document its usage ？

Reply:
+1 for allowing user setting this explicitly.  seems no need to add `jdbc.schema-version` in IcebergConfig, do you like to add related document in this PR or a separate PR?